Formula/jless.rb

```diff
@@ -0,0 +1,17 @@
+class Jless < Formula
+  desc "Command-line pager for JSON data"
+  homepage "https://pauljuliusmartinez.github.io/"
+  url "https://github.com/PaulJuliusMartinez/jless/archive/refs/tags/v0.7.1.tar.gz"
+  sha256 "8b36d117ac0ef52fa0261e06b88b5ae77c2ff4beeb54f2c906a57f066dc46179"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/jless", "--help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

You should probably be able to interact with this using Ruby's PTY module.

Reply:
Ah, sorry! Will figure that out and add a better test.

Reply:
Couldn't figure out how to get the PTY module working with the interactive fullscreen thing this program does, but it also has a non-interactive mode that's easy to test by redirecting the output, so I've just tested that instead.